app/views/admin/orders/index.html.erb

```diff
@@ -1,5 +1,7 @@
 <h1>All Orders:</h1>
 
+<%= link_to 'Download all orders as CSV', admin_orders_export_path(format: "csv"), class: 'btn btn-outline-success btn-sm' %>
```

Comment:
Personally I would not send a parameter here, since we can't export anything else than csv anyway - but this is only my preference (not an error)
```suggestion
<%= link_to 'Download all orders as CSV', admin_orders_export_path, class: 'btn btn-outline-success btn-sm' %>
```